tpchgen/src/distribution.rs

```diff
@@ -217,28 +159,118 @@ impl DistributionLoader {
 /// Static global instance of the default distributions.
 ///
 /// Initialized once on first access.
-static DEFAULT_DISTRIBUTIONS: LazyLock<Distributions> = LazyLock::new(Distributions::default);
+static DEFAULT_DISTRIBUTIONS: LazyLock<Distributions> =
+    LazyLock::new(|| Distributions::try_load_default().unwrap());
 
 /// Distributions wraps all TPC-H distributions and provides methods to access them.
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, Default)]
 pub struct Distributions {
-    distributions: IndexMap<String, Distribution>,
+    articles: Distribution,
```

Comment:
They key change is that now the distributions are loaded into the correct field at load time, rather than on access